app/ui/webpack.config.hot.js

```diff
@@ -0,0 +1,23 @@
+var config = require('./webpack.config.base');
```

Comment:
Surely you're using a new enough version of node to use `let`, it's been around since ~v0.10